src/pages/projectSettings/tabs/ViewsAndFiltersTab/getFormSchema.ts

```diff
@@ -2,44 +2,15 @@ import { GetFormSchema } from "components/SpruceForm";
 import { CardFieldTemplate } from "components/SpruceForm/FieldTemplates";
 import widgets from "components/SpruceForm/Widgets";
 import { ProjectHealthView } from "gql/generated/types";
+import { ProjectType } from "../utils";
 
```

Comment:
Should I be able to see the repo filters on Parsley in the project filters modal? I'm testing using localhost, but I don't see them... :o

Reply:
Thank you, I opened an Evergreen PR to address this: https://github.com/evergreen-ci/evergreen/pull/7200